generators/typescript/utils/core-utilities/fetcher/src/form-data-utils/__test__/multipart.upload.test.ts

```diff
@@ -0,0 +1,62 @@
+import express from "express";
```

Comment:
Here's a link to [prism](https://github.com/stoplightio/prism?tab=readme-ov-file#mocking), we'll want to use that since it handles all of the checking around multipart files + provides really good logs (as opposed to building file support in fern). 

Open to other options here though, mostly just optimizing for speed.